lib/eventum/class.mime_helper.php

```diff
@@ -177,13 +177,17 @@ public static function encodeAddress($address)
         if (self::is8bit($address)) {
             // split into name and address section
             preg_match('/(.*)<(.*)>/', $address, $matches);
+
+            $qq = preg_replace_callback(
+                '/([\x80-\xFF]|[\x21-\x2F]|[\xFC]|\[|\])/',
+                function ($m) {
+                    return '=' . strtoupper(dechex(ord(stripslashes($m[1]))));
```

Comment:
i didn't bother to dig into what it does. i just converted syntax. it worked so far, and idea is to drop those methods when finally able to switch to zend mail classes.